api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
@ncdc You are right. I verified that exec expects multiple args of the same name (multiple command query params, in this case) of the form: exec?command=echo&command=ABC

@wing328 Do you know whats the best way to represent that?